src/App.tsx

```diff
@@ -9,33 +10,39 @@ function App() {
     first: "",
     last: "",
   });
-  const [trendingRecipes, setTrendingRecipes] = useState<Trending>()
+  const [trendingRecipes, setTrendingRecipes] = useState<Trending[]>()
+  const [errorMessage, setErrorMessage] = useState<string | null>(null)
 
   const handleNameUpdate=(field: keyof typeof name, newName: string) => {
-    setName((prevState) => {
-      prevState[field] = newName
-      return prevState
-    })
+    setName(prevState => ({
+      ...prevState,
+      [field]: newName
+    }))
   }
 
   useEffect(() => {
     const fetchTrending = async () => {
-      const trending = await getTrending()
-      setTrendingRecipes(trending)
+      try {
+        const trending = await getTrending()
+        setTrendingRecipes(trending)
+        setErrorMessage(null)
+      } catch (error) {
+        console.error('Error in fetching trending data: ', error)
```

Comment:
A potential error message on this console could be "Error in fetching trending data: Error in fetching trending data: ...status" since you have the error message format set up already in the api function try catch.